### ISSUE TITLE
Session files are not portable

### ISSUE THREAD

**Issue by bennyevangelista** [zarunbal/LogExpert#269]
Moving to a new workstation I copied all my .lxj files but the don't work. When I try to open them I see a lots of overlapping empty windows and no log is opened.

I opened a lxj file with Notepad++ and noticed that there are references to the absolute paths of .lxp files (didn't know about them) that are missing in my new workstation.

I tried to copy the lxp files too, and updated the paths in lxp file, but it's not working. It seems to work only if I first open a log file alone, and then I open the session that references that file.

All the logs are on network paths, I'm using LogExpert 1.8.7.

**Comment by Hirogen** [comment]
Could you test this with 1.9.0, this should have been fixed in Version 1.9.0 :)

**Comment by bennyevangelista** [comment]
Just tried, same behaviour. Do I have to copy the lxp files too?

**Comment by Hirogen** [comment]
> Just tried, same behaviour. Do I have to copy the lxp files too?

yes, they are session files, if you change the configuration in the middle, session files are not copied automatically

**Comment by bennyevangelista** [comment]
ok, after copying lxp files whit 1.9.0 session files work again. 

Here's my doubt: as a "user" I don't know about the existance of lxp files and I've always thought of lxj files as a "list" of log files. Why LogExpert doesn't just open these log files recreating any not found lxp file? This would make migration and environment setup much easier :)

**Comment by Hirogen** [comment]
> ok, with 1.9.0 and after copying lxp files, session files work again.
> 
> Here's my doubt: as a "user" I don't know about the existance of lxp files and I've always thought of lxj files as a "list" of log files. Why LogExpert doesn't just open these log files recreating any not found lxp file? This would make migration and environment setup much easier :)

because those are session files for a specific logfile, where session data is saved, sadly you can't just "recreate" the data 😄

**Comment by bennyevangelista** [comment]
> because those are session files for a specific logfile, where session data is saved, sadly you can't just "recreate" the data 😄

My bad, I can't make myself clear. I mean that since in the lxj files is stored the list of log files in the session (and the related lxp files), when the lxp files are missing couldn't LogExpert just open the listed log files and create the missing lxps from scratch (default settings), as it is opening them for the first time?

**Comment by Hirogen** [comment]
this has been implemented, you can now selected missing files and choose there respective folders if logexpert does not find them.

This has not been added to the wiki, and still needs an entry

<img width="1257" height="961" alt="Image" src="https://github.com/user-attachments/assets/1b2edcc7-bba3-4d42-bff4-0ccb75d0b092" />
